e2e/thin_provisioning_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"os/exec"
```

Comment:
I've struggled with the mystery of number of diff lines (+547 −548) and finally found this removal. Yay!